### PR TITLE
Add project Popup for desktop and mobile version

### PR DESCRIPTION
**Pull request by harshitajain06**
🚨Please review the to this repository.

### Description
Added Popup window for project description using HTML, CSS and Javascript.

💔Thank you!

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,19 +1,126 @@
...
+respMenu.addEventListener('click', () => {
+  respMenu.style.display = 'none';
+});
 
-function close1() {
-    respMenu.style.display = "none";
-}
+// **************Popup*******************
+const projects = [
+  {
```
Comment (tufoinnkuo10) [inline main.js:22]: - [ ] great job. please ensure that when the popup comes up the user should not be able to scroll through your portflolio unless he/she closes it.

File: main.js [hunk 2]
```diff
@@ -1,19 +1,126 @@
...
+    title: 'Project name goes here',
+    img: './images/Group 105.svg',
+    imgAlt: 'First img',
+    tech: ['HTML/CSS', 'Ruby on Rails', 'Java'],
+    des: ['Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.', ' Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.'],
+    git: ['#', 'See Github'],
+    demo: ['#', 'See Demo'],
```
Comment (tufoinnkuo10) [inline main.js:29]: - [ ] kindly maintain the color of the see source button like that of the figma version in the mobile version
yours
Screenshot (589)_LI
figma
Screenshot (588)

File: main.js [hunk 3]
```diff
@@ -1,19 +1,126 @@
...
+  },
+  {
+    title: 'Project name goes here',
+    img: './images/Rectangle 21 (1).svg',
```
Comment (tufoinnkuo10) [inline main.js:33]: - [ ] please in other to follow the figma design, align your `previous `and `see live` button so it follow the text
yours
Screenshot (590)_LI
figma
Screenshot (587)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @harshitajain06 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ShahierNashaat — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**